yarn.lock

```diff
@@ -5602,21 +5602,21 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/bridge-controller@npm:^37.0.0":
-  version: 37.0.0
-  resolution: "@metamask/bridge-controller@npm:37.0.0"
+"@metamask/bridge-controller@npm:39.0.0":
+  version: 39.0.0
+  resolution: "@metamask/bridge-controller@npm:39.0.0"
   dependencies:
     "@ethersproject/address": "npm:^5.7.0"
     "@ethersproject/bignumber": "npm:^5.7.0"
     "@ethersproject/constants": "npm:^5.7.0"
     "@ethersproject/contracts": "npm:^5.7.0"
     "@ethersproject/providers": "npm:^5.7.0"
-    "@metamask/base-controller": "npm:^8.0.1"
+    "@metamask/base-controller": "npm:^8.1.0"
     "@metamask/controller-utils": "npm:^11.11.0"
     "@metamask/gas-fee-controller": "npm:^24.0.0"
-    "@metamask/keyring-api": "npm:^19.0.0"
+    "@metamask/keyring-api": "npm:^20.0.0"
```

Comment:
I wonder why "@metamask/keyring-api" is not set to "^19.1.0"

Reply:
When you use a resolution to override a dependency, the original requested dependency remains in the lockfile here. But it gets _resolved_ by Yarn differently. Essentially these listenings under "dependencies:" are what gets overwritten when we make a resolution.

Reply:
This explains all three questions I believe

Reply:
Ok thanks for explanation